main.go

```diff
@@ -66,9 +84,12 @@ func main() {
 			}
 		}
 
-		contentLineSplit[0] = bytes.Join(output, delimiter)
-
-		_, _ = os.Stdout.Write(bytes.Join(contentLineSplit, []byte{'\n'}))
+		if *fieldsParameter != "" {
+			contentLineSplit[0] = bytes.Join(output, delimiter)
+			_, _ = os.Stdout.Write(bytes.Join(contentLineSplit, []byte{'\n'}))
+		} else if *charParameter != "" {
+			fmt.Printf("%s\n", bytes.Join(output, []byte("")))
+		}
```

Comment:
What's this for?

Reply:
to print characters and not words

Reply:
Do you actually need this change?

Reply:
well its not working with just stdout.write and its the qucikest solution ive found

Reply:
If I revert this change, it works for me.

Reply:
![Screenshot from 2020-05-13 12-19-28](https://user-images.githubusercontent.com/55092455/81801010-0987ca00-9514-11ea-85d2-6140eb9eef01.png)
it does work but it looks ugly

Reply:
```
➜  trainee-go-cut git:(feature/parameter-c-3) ✗ git diff
diff --git a/main.go b/main.go
index 14ce070..f103d8a 100644
--- a/main.go
+++ b/main.go
@@ -84,12 +84,12 @@ func main() {
                        }
                }

-               if *fieldsParameter != "" {
+               //if *fieldsParameter != "" {
                        contentLineSplit[0] = bytes.Join(output, delimiter)
                        _, _ = os.Stdout.Write(bytes.Join(contentLineSplit, []byte{'\n'}))
-               } else if *charParameter != "" {
+               /*} else if *charParameter != "" {
                        fmt.Printf("%s\n", bytes.Join(output, []byte("")))
-               }
+               }*/

                if readAllErr == io.EOF {
                        break
➜  trainee-go-cut git:(feature/parameter-c-3) ✗ go run . -c 2-3
abcd
bc
➜  trainee-go-cut git:(feature/parameter-c-3) ✗
```

Reply:
it has something to do with the OS then

Reply:
If you make exactly my above changes and nothing more – doesn't it work for you?

Reply:
if i revert the changes i made it looks like on the picture

Reply:
What does `hexdump -C < toCut.txt` say?

Reply:
```
00000000  61 09 62 09 63 0a 64 09  65 09 66                 |a.b.c.d.e.f|
0000000b

```